package.json

```diff
@@ -119,7 +119,7 @@
     "husky": "1.0.0-rc.8",
     "jasmine-core": "2.99.1",
     "jasmine-expect": "3.8.3",
-    "jasmine-spec-reporter": "4.2.1",
+    "jasmine-spec-reporter": "4",
```

Comment:
How does this version work!?

Reply:
You can put any valid tag or semver version, which will be resolved here as anything 4.x. We should fix this though to be specific.

Reply:
yeah, I missed putting that back when I was looked at what testing deps could also be upgraded. I'll revert and submit a separate commit for testing deps if they are applicable.